packages/web/src/components/nav/desktop/NavMenuButton.tsx

```diff
@@ -52,6 +55,7 @@ export const NavMenuButton = () => {
   const isUSDCEnabled = useIsUSDCEnabled()
   const challengeRewardIds = useRemoteVar(StringKeys.CHALLENGE_REWARD_IDS)
   const hasClaimableRewards = useAccountHasClaimableRewards(challengeRewardIds)
+  const isManagedAccount = useIsManagedAccount()
```

Comment:
It occurs to me that this is going to return false until it finishes loading both accounts. So it's possible we might get flashes if navigating directly to a route. Could be solved, but maybe we do that as a follow up.